doc/website/getting-started/overview.md

```diff
@@ -11,637 +10,242 @@ runtime and create _publishers_ and _subscribers_. The publishers send data of a
+Afterwards, we can start the applications which immediately connect to the RouDi via their runtime.
 
-This only allows us to get one sample at a time. Should we want to get all currently available samples we can do so
-by using an additional loop.
+When the application terminates, the runtime cleans up all resources needed for communication with RouDi. This
+includes all memory chunks used for the data transmission which may still be held by the application.
 
-```cpp
-while (keepRunning)
-{
-    while (subscriber.hasData())
-    {
-        subscriber->take()
-            .and_then([](iox::popo::Sample<const CounterTopic>& sample) {
-                CounterTopic* ptr = sample.get();
-                /* process the received data using the ptr */
-
-                /* alternatively use operator-> */
-                uint32_t counter = sample->counter;
-            })
-            .or_else([](iox::popo::ChunkReceiveResult) { /* handle the error */ });
-    }
-    // wait for new samples (either sleep or use the waitset)
-}
-```
 
-Here we do not check whether we actually have data since we already know there is data available by calling `hasData`.
+We now briefly define the main entities of an iceoryx system which were already used in the example above.
 
-### Untyped API
+## RouDi
 
-The untyped API offeres similar capabilities and is hence usable in a similar way. The major difference is that
-neither publisher nor subscriber have any knowledge about the underlying type they send or receive. This means that
-the user is responsible to ensure the data is read correctly, i.e. there is no type safety guaranteed by the API
-itself.
+RouDi is an abbreviation for **Rou**ting and **Di**scovery. RouDi takes care of the
+communication setup but does not actually participate in the communication between the publisher and the subscriber.
+RouDi can be thought of as the switchboard operator of iceoryx. One of its other major tasks is the setup of the
+shared memory, which the applications use for exchanging payload data. Sometimes referred to as daemon, RouDi manages
+the shared memory and is responsible for the service discovery, i.e. enabling subscribers to find topics offered by 
+publishers. It also keeps track of all applications which have initialized a runtime and are hence able to use
+publishers or subscribers. To view the available command line options call `$ICEORYX_ROOT/build/iox-roudi --help`.
 
-#### Creating a publisher
+## Runtime
 
-When creating an untyped publisher we do not need to specify a data type as template paraemter.
+Each application that wants to use iceoryx has to instantiate its runtime, which essentially enables communication
+with RouDi. Only one runtime object per user process is allowed.
 
+To do so, the following lines of code are required
 ```cpp
-iox::popo::UntypedPublisher publisher({"Group", "Instance", "CounterTopic"});
-publisher.offer();
+iox::runtime::PoshRuntime::initRuntime("some_unique_application_name");
```

Comment:
What does the runtime do? Why do we need to create it? Please mention here, that the runtime abstracts the communication to RouDi, it also maps the shared memory to the apps address space, etc.